projects/benqi-ignite/package.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "@projects/benqi-ignite",
+    "version": "1.0.0",
+    "scripts": {
+        "yarn": "yarn install",
+        "test": "vitest run",
+        "test:watch": "vitest"
+    },
+    "dependencies": {
+        "@heyanon/sdk": "^1.0.4"
+    },
+    "license": "MIT",
+    "engines": {
+        "npm": "please-use-yarn",
```

Comment:
😂 

Wondering if that would be the case for other integrations over this repo 🙃

Reply:
Same as https://github.com/RealWagmi/anon-integration-guide/pull/16#discussion_r1935229131